assets/app/views/catalog.html

```diff
@@ -0,0 +1,51 @@
+<div ng-controller="CatalogController">
+  <h1>Components</h1>
+  <section class="component-search">
+    <label>Search for templates or images</label>
+    <input ng-model="searchText" type="text" class="search" />
+  </section>
+  <section class="template-list">
+    <ul>
+      <li>
+        <templates list="myTemplates" title="'My Templates'"></templates>
+      </li>
+      <li>
+        <templates list="myOrgTemplates" title="myorgTitle + ' Templates'"></templates>
+      </li>
+        <li ng-include="views/_custom_create.html">
+      </li>
+    </ul>
+  </section>
+  <section class="template-list">
+    <ul>
+      <li>
+        <templates list="featured[0].list" title="featured[0].title"></templates>
+      </li>
+      <li>
+        <templates list="featured[1].list" title="featured[1].title"></templates>
+      </li>
+      <li>
+        <templates list="features[2].list" title="featured[2].title"></templates>
```

Comment:
And probably we're going to dramatically change this anyway.  Make this a loop and show all of them - we'll sort through what it should look later